core/src/test/java/com/gentics/mesh/core/schema/field/FieldMigrationTestcases.java

```diff
@@ -11,13 +11,6 @@
 	 */
 	void testRemove() throws Exception;
 
-	/**
-	 * Test renaming the field
-	 * 
-	 * @throws Exception
-	 */
-	void testRename() throws Exception;
```

Comment:
Check why this was removed.

Reply:
A custom script was used for renaming fields.